nodejs/week3/app.js

```diff
@@ -0,0 +1,67 @@
+// @ts-ignore
+const knex = require('knex')({
+  client: 'mysql2',
+  connection: {
+    host: process.env.DB_HOST || '0.0.0.0',
+    port: process.env.DB_PORT || 33060,
+    user: process.env.DB_USER || 'root',
+    password: process.env.DB_PASSWORD || 'pass',
+    database: process.env.DB_NAME || 'hyf_node_week3_warmup',
+    multipleStatements: true,
+  },
+});
+
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+
+const apiRouter = express.Router();
+app.use('/api', apiRouter);
+
+const contactsAPIRouter = express.Router();
+apiRouter.use('/contacts', contactsAPIRouter);
+
+contactsAPIRouter.get('/', async (req, res) => {
+  let query = knex.select('').from('contacts');
+
+  if ('sort' in req.query) {
+    // @ts-ignore
+    const orderBy = req.query.sort.toString();
+    if (orderBy.length > 0) {
+      query = query.orderByRaw(orderBy);
+    }
+  }
+
+  console.log('SQL', query.toSQL().sql);
```

Comment:
Its okay to have console.log but when a merge PR is raised - it believe it should be removed.